libgloss/arc64/arc_main.c

```diff
@@ -0,0 +1,45 @@
+/*********************************************************************
```

Comment:
Please sync the relevant files with ARCv1/v2 commit.

Reply:
done